lib/modules/dosomething/dosomething_api/includes/Transformer.php

```diff
@@ -329,10 +331,13 @@ protected function transformMedia($data, $aspect_ratio = NULL) {
    * @return array
    */
   protected function transformReportback($data) {
+    $iso_created_at = date('c', $data->created_at);
+    $iso_updated_at = date('c', $data->updated_at);
+
     $output = [
       'id' => $data->id,
-      'created_at' => $data->created_at,
-      'updated_at' => $data->updated_at,
+      'created_at' => $iso_created_at,
+      'updated_at' => $iso_updated_at,
```

Comment:
instead of creating temp vars here,  you could just set it
`  'created_at' => date('c', $data->created_at)`